### PR TITLE
Enhance login button of google an facebook

### PR DESCRIPTION
**Pull request by BlackWolfie**
Add new design of the google and facebook login button

### CODE REVIEW

**Comment by BlackWolfie** [comment]
image

File: editor/src/app/login/login.component.ts [hunk 1]
```diff
@@ -27,9 +27,11 @@ import { PopupService } from '../popup/popup.service';
     <ng-template #logInForm>
       <div *ngIf="appState.isBertaHosting" class="form-group social-login">
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=facebook" class="button facebook">
-          Log in with Facebook</a>
+        <bt-icon-facebook></bt-icon-facebook>
+        <p>Log in with Facebook</p></a>
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=google" class="button google">
-          Log in with Google</a>
+        <img src="../../../../engine/layout/Google.png" alt="G">
```
Comment (vteivans) [inline editor/src/app/login/login.component.ts]: Check how other images are referenced in the login component. See line [21](https://github.com/berta-cms/berta/pull/300/files#diff-7f57b5386344a10b7e842dc23fe57536R21).

File: editor/src/styles/_login.scss [hunk 2]
```diff
@@ -34,19 +37,61 @@ berta-login {
...
+        }
+
         &:hover {
           background-color: darken($facebook-color, 10%);
         }
       }
 
-      &.google {
-        background-color: $google-color;
+      &.google:enabled {
```
Comment (vteivans) [inline editor/src/styles/_login.scss]: Why do you have `:enabled` state? The normal state is enabled - that's the default one.

File: editor/src/app/login/login.component.ts [hunk 3]
```diff
@@ -27,9 +27,11 @@ import { PopupService } from '../popup/popup.service';
     <ng-template #logInForm>
       <div *ngIf="appState.isBertaHosting" class="form-group social-login">
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=facebook" class="button facebook">
-          Log in with Facebook</a>
+        <bt-icon-facebook></bt-icon-facebook>
```
Comment (uldisrudzitis) [inline editor/src/app/login/login.component.ts]: I'm not sure if need to extract it as component if it's simply a piece of svg code and not even used somewhere else.

File: editor/src/app/login/login.component.ts (hunk 1 above)
Comment (uldisrudzitis) [inline editor/src/app/login/login.component.ts]: Would be nice to use SVG here as well.

Comment (uldisrudzitis) [reply editor/src/app/login/login.component.ts]: https://developers.google.com/identity/branding-guidelines

File: editor/src/app/login/login.component.ts (hunk 3 above)
Comment (BlackWolfie) [reply editor/src/app/login/login.component.ts]: @vteivans said it's good to use a component here

Comment (uldisrudzitis) [reply editor/src/app/login/login.component.ts]: No problem, let's be consistent and use component for google as well.

**Comment by BlackWolfie** [comment]
image

File: editor/src/app/app.module.ts [hunk 4]
```diff
@@ -37,7 +37,7 @@ import { SentryErrorHandler } from './sentry.error-handler';
     UserAccountComponent,
     LoginComponent,
     PreviewComponent,
-    PopupComponent
+    PopupComponent,
```
Comment (uldisrudzitis) [inline editor/src/app/app.module.ts]: Please revert this change

File: editor/src/app/shared/shared.module.ts [hunk 5]
```diff
@@ -1,14 +1,20 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { LoadingComponent } from './loading/loading.component';
-
+import { IconGoogleComponent } from './icon-google.component';
+import { IconFacebookComponent } from './icon-facebook.component';
 @NgModule({
-  declarations: [LoadingComponent],
+  declarations: [LoadingComponent,
```
Comment (uldisrudzitis) [inline editor/src/app/shared/shared.module.ts]: Put the `LoadingComponent` on new line

File: editor/src/styles/_login.scss [hunk 6]
```diff
@@ -44,9 +62,36 @@ berta-login {
...
+          justify-content: center
         }
+
+        & svg {
+          width: 20px;
+        }
+
+        & p {
+          margin-top: 0;
+          font-size: 18px;
```
Comment (uldisrudzitis) [inline editor/src/styles/_login.scss]: Font size should be `1rem` like all other buttons in interface.

File: editor/src/styles/_login.scss [hunk 7]
```diff
@@ -44,9 +62,36 @@ berta-login {
       &.google {
```
Comment (uldisrudzitis) [inline editor/src/styles/_login.scss]: Please DRY your css. Add only specific css properties in `google` and `facebook` classes. All other common properties should be shared from `social-login a`.

File: editor/src/styles/_login.scss [hunk 8]
```diff
@@ -63,21 +111,22 @@ berta-login {
...
+      text-align: left;
     }
 
     button {
       display: block;
       font-size: $base-font-size;
       margin-bottom: 1.3em;
-      margin-left: auto;
-      width: 50%;
+      margin-right: auto;
```
Comment (vteivans) [inline editor/src/styles/_login.scss]: you can remove this margin now, since the element is 100% wide anyway
